### ISSUE TITLE
nix-shell completion is broken with default macOS zsh 5.3

### ISSUE THREAD

**Issue by chuwy** [spwhitt/nix-zsh-completions#27]
### Description

Tab-completions for `nix-shell` throws:

```
_arguments:comparguments:319: invalid argument: +
```

Right after I press the tab button.

It works fine however if I use zsh 5.7.1 installed by nix. I don't know if we consider this an issue, given that it is a problem exists only with an old version (which is however very popular).

### System

- nix-zsh-completions version: adbf7bf
- ZSH version: 5.3
- NIX_PATH: /Users/antonparkhomenko/.nix-defexpr/channels:/Users/antonparkhomenko/.nix-defexpr/channels

- system: `"x86_64-darwin"`
- host os: `Darwin 18.7.0, macOS 10.14.6`
- multi-user?: `no`
- sandbox: `no`
- version: `nix-env (Nix) 2.3.1`
- channels(antonparkhomenko): `"nixpkgs-20.03pre205792.fb1bc1b891f"`
- nixpkgs: `/Users/antonparkhomenko/.nix-defexpr/channels/nixpkgs`

**Comment by hedning** [comment]
Right, managed to miss this one. Pushed 23cbcf53fa259a23bb56d3641c70cc815f8e9b8f which should fix it.

**Comment by Mic92** [comment]
Still a problem or can it be closed?

**Comment by chuwy** [comment]
Not a problem for me at least. I fixed it by switching everying to Nix-provided ZSH.

**Comment by hedning** [comment]
As far as I know https://github.com/spwhitt/nix-zsh-completions/commit/23cbcf53fa259a23bb56d3641c70cc815f8e9b8f fixes it, but I need to do a new release.
